README.md

```diff
@@ -94,6 +94,51 @@ module.exports = {
 };
 ```
 
+### Additional Scenarios
+Scenario 1
+> Alice is using a CSS Extractor and her images are not loading successfully. In the loader, she should apply the ```encoding``` parameter with a value of ```'base64'``` to ensure quotes are added to her url(...) call.
+``` javascript
+module.exports = {
+  //...
+	module: {
+		rules: [
+			{
+			    test: /\.svg/,
+			    use: {
+			        loader: 'svg-url-loader',
+			        options: {
+				          encoding: 'base64'
+		        	}
+			    }
+			}
+		]
+	},
+	//...
+};
+```
+
+Scenario 2
+> John needs to support IE and is seeing black fill-color for his styled shapes. In the loader, he should apply the ```iesafe``` parameter with a value of ```true``` to default back to the file-loader.
```

Comment:
please change to read `... with a value of ```true``` to use file-loader for images IE cannot handle (bigger than 4KB).`